commit 5be255c28b24944b530bb8f91e6bde2e66992c2a
Author: Joseph Myers <joseph@codesourcery.com>
Date:   Tue Sep 1 13:45:49 2015 +0000

    Add more TCP_* values to netinet/tcp.h.
    
    This patch adds move TCP_* values to sysdeps/gnu/netinet/tcp.h to
    bring it up to date with Linux 4.2.  TCP_SAVE_SYN and TCP_SAVED_SYN
    are new in 4.2, TCP_NOTSENT_LOWAT and TCP_CC_INFO are older (Szabolcs
    previously noted in
    <https://sourceware.org/ml/libc-alpha/2015-06/msg00938.html> that this
    header was out of date in glibc).
    
    Tested for x86_64 (testsuite, and that installed stripped shared
    libraries are unchanged by the patch).
    
    	* sysdeps/gnu/netinet/tcp.h (TCP_NOTSENT_LOWAT): New macro.
    	(TCP_CC_INFO): Likewise.
    	(TCP_SAVE_SYN): Likewise.
    	(TCP_SAVED_SYN): Likewise.

diff --git a/sysdeps/gnu/netinet/tcp.h b/sysdeps/gnu/netinet/tcp.h
index f6602ec..3918bca 100644
--- a/sysdeps/gnu/netinet/tcp.h
+++ b/sysdeps/gnu/netinet/tcp.h
@@ -61,6 +61,14 @@
 #define TCP_REPAIR_OPTIONS	 22 /* Repair TCP connection options */
 #define TCP_FASTOPEN		 23 /* Enable FastOpen on listeners */
 #define TCP_TIMESTAMP		 24 /* TCP time stamp */
+#define TCP_NOTSENT_LOWAT	 25 /* Limit number of unsent bytes in
+				       write queue.  */
+#define TCP_CC_INFO		 26 /* Get Congestion Control
+				       (optional) info.  */
+#define TCP_SAVE_SYN		 27 /* Record SYN headers for new
+				       connections.  */
+#define TCP_SAVED_SYN		 28 /* Get SYN headers recorded for
+				       connection.  */
 
 #ifdef __USE_MISC
 # include <sys/types.h>
